mriqc/reports/generators.py

```diff
@@ -176,14 +178,14 @@ def summary_cover(dframe, qctype, failed=None, sub_id=None, out_file=None):
     # Format the size
     #pylint: disable=E1101
     newdf[['size_x', 'size_y', 'size_z']] = newdf[['size_x', 'size_y', 'size_z']].astype(np.uint16)
-    formatter = lambda row: ur'%d \u00D7 %d \u00D7 %d' % (
```

Comment:
Is there a reason you want them to stay as a raw string though? In python 3 (and in py2 with the unicode_literals import), it should handle the unicode characters with the backslash correctly without having to make it a raw string